docs/manual/format_lead.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: rpm.org - RPM Package Lead format
+---
+
+## Lead Format
+
+The Lead is stored as a C structure:
+
+```
+struct rpmlead {
+    unsigned char magic[4];
+    unsigned char major, minor;
+    short type;
+    short archnum;
+    char name[66];
+    short osnum;
+    short signature_type;
+    char reserved[16];
+};
+```
```

Comment:
Should endian-ness be discussed here?

Reply:
Right, now that its a standalone document somebody could read it without first coming up with the "all integer data is in network order" mention in the package format doc(s).